docs/proposals/20-ha-control-planes.md

```diff
@@ -365,6 +366,21 @@ With most of the complexity being handled by [Etcd-Druid](https://github.com/gar
 
 The groundwork for this was already done by [gardener/gardener#5741](https://github.com/gardener/gardener/pull/5741).
 
+### Other critical components having single replica
+
+Following shoot control plane components are currently setup with a single replica and are planned to run with a minimum of 2 replicas:
+
+* Cluster Autoscaler (if enabled)
```

Comment:
I was also thinking about the CSI drivers (the controller plugins). They are also kind of critical component as they do the attach/detach operations. Can you elaborate why the CSI drivers are not this list?

Reply:
Added that now. This was missed.